CMakeLists.txt

```diff
@@ -157,71 +157,71 @@ find_package(Patch REQUIRED)
-		-DCMAKE_FIND_ROOT_PATH=${CMAKE_FIND_ROOT_PATH}
-		-DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=${CMAKE_FIND_ROOT_PATH_MODE_PROGRAM}
-		-DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=${CMAKE_FIND_ROOT_PATH_MODE_LIBRARY}
-		-DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=${CMAKE_FIND_ROOT_PATH_MODE_INCLUDE}
-		-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
-		-DCOMPILE_DEFINITIONS=${COMPILE_DEFINITIONS}
-		-DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
-		-DCMAKE_CXX_STANDARD_REQUIRED=${CMAKE_CXX_STANDARD_REQUIRED}
-		-G${CMAKE_GENERATOR}
-		)
+    -DANDROID_PLATFORM=${ANDROID_PLATFORM}
+    -DANDROID_STL=${ANDROID_STL}
+    -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    -DANDROID_NDK=${ANDROID_NDK}
+    -DCMAKE_POSITION_INDEPENDENT_CODE=${CMAKE_POSITION_INDEPENDENT_CODE}
+    -DCMAKE_C_FLAGS=${PASSTHROUGH_CMAKE_C_FLAGS}
+    -DCMAKE_CXX_FLAGS=${PASSTHROUGH_CMAKE_CXX_FLAGS}
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
+    -DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}
+    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+    -DCMAKE_EXE_LINKER_FLAGS=${CMAKE_EXE_LINKER_FLAGS}
+    -DCMAKE_SHARED_LINKER_FLAGS=${CMAKE_SHARED_LINKER_FLAGS}
+    -DCMAKE_FIND_ROOT_PATH=${CMAKE_FIND_ROOT_PATH}
+    -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=${CMAKE_FIND_ROOT_PATH_MODE_PROGRAM}
+    -DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=${CMAKE_FIND_ROOT_PATH_MODE_LIBRARY}
+    -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=${CMAKE_FIND_ROOT_PATH_MODE_INCLUDE}
+    -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+    -DCOMPILE_DEFINITIONS=${COMPILE_DEFINITIONS}
+    -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
+    -DCMAKE_CXX_STANDARD_REQUIRED=${CMAKE_CXX_STANDARD_REQUIRED}
+    -G${CMAKE_GENERATOR}
+    )
 
 # jemalloc
 if(NOT WIN32)
-	if (ENABLE_JNI)
-		if (NOT DISABLE_JEMALLOC)
-			include(BundledJemalloc)
-			use_bundled_jemalloc(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-			message("jemalloc found at ${JEMALLOC_LIBRARIES}")
-		endif()
-		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_JNI")
-		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DENABLE_JNI")
-	endif()
+    if (ENABLE_JNI)
+        if (NOT DISABLE_JEMALLOC)
+            include(BundledJemalloc)
+            use_bundled_jemalloc(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+            message("jemalloc found at ${JEMALLOC_LIBRARIES}")
+        endif()
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_JNI")
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DENABLE_JNI")
+    endif()
 endif()
 
 # Simple-Windows-Posix-Semaphore
 if (WIN32)
-	add_subdirectory("thirdparty/Simple-Windows-Posix-Semaphore")
+     add_subdirectory("thirdparty/Simple-Windows-Posix-Semaphore")
 endif()
 
 # ossp-uuid
 if(NOT WIN32)
-	include(BundledOSSPUUID)
-	use_bundled_osspuuid(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+    include(BundledOSSPUUID)
+     use_bundled_osspuuid(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
One extra space here

Reply:
Updated in f5fca2273376524f4566f9a5044d419f3e7fa5b3